lib/fontist/manifest/locations.rb

```diff
@@ -5,8 +5,21 @@ def initialize(manifest)
         @manifest = manifest
       end
 
-      def self.call(manifest)
-        new(manifest).call
+      def self.call(file, **keywords)
```

Comment:
It would make more sense if this is called `from_file` rather than `call` since it's expecting a YAML file.

Reply:
Yes, it would make more sense. Renamed :)